test/Libraries/PackageManagerTests/PackageLoaderTests.cs

```diff
@@ -905,7 +1029,11 @@ public void LocalizedPackageLocalizedCorrectly()
             Thread.CurrentThread.CurrentCulture = esculture;
             Thread.CurrentThread.CurrentUICulture = esculture;
 
-            var loader = new PackageLoader(new[] { PackagesDirectory }, new[] {string.Empty});
+            var pathManager = new Mock<Dynamo.Interfaces.IPathManager>();
+            pathManager.SetupGet(x => x.PackagesDirectories).Returns(() => new[] { PackagesDirectory });
```

Comment:
was this a necessary change for some reason or just a nice cleanup unifying tests?

Reply:
Yes, this is a cleanup to unify tests. I do not want to continue using the old `PackageLoader` constructors and wish to use the new ones that take in a `PathManager` instance instead so as to set a good example of writing new tests. This will also make it clearer as to which old constructors can be deprecated in the code.